sdk/identity/identity/CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 ## 1.1.0 (2020-08-11)
 
+### Changes since 1.0.*
+
+- With 1.1.0, new developer credentials are now available: `VisualStudioCodeCredential` and `AzureCLICredential`. These developer credentials may be used directly or indirectly as part of `DefaultAzureCredential`.
```

Comment:
Can we have sub bullets with one liners on what these new credentials are? Or are we saying that the name is self explanatory? Am ok either way